node/pkg/reporter/attestation_events_test.go

```diff
@@ -0,0 +1,34 @@
+package reporter
+
+import (
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"testing"
+)
+
+func TestEventListener(t *testing.T) {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
+
+	subs_expect := map[int]*lifecycleEventChannels{}
+
+	attestationEventReporter := EventListener(logger)
+	assert.Equal(t, subs_expect, attestationEventReporter.subs)
+}
+
+func TestGetUniqueClientId(t *testing.T) {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
+
+	attestationEventReporter := EventListener(logger)
+	assert.Equal(t, 498081, attestationEventReporter.getUniqueClientId())
+}
+
+func TestSubscribe(t *testing.T) {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
+
+	attestationEventReporter := EventListener(logger)
+	activeSubscription := attestationEventReporter.Subscribe()
+	assert.Equal(t, 727887, activeSubscription.ClientId)
```

Comment:
Ditto as above. Mind making that number a variable that makes it more obvious what it is or why it is that specific number?

Reply:
Once https://github.com/certusone/wormhole/pull/1127 merges, I'll readjust this to reflect the new behavior.